mysql-test/include/gap_lock_error_cleanup.inc

```diff
@@ -0,0 +1,6 @@
+
+drop table if exists gap1, gap2, gap3, gap4;
+
+--exec grep "Got error" $MYSQLTEST_VARDIR/log/mysqld.1.err | grep "gap" | wc -l
```

Comment:
Makes sense, this was copied/inherited from the upstream test and I didn't think much of it. I might report it to them for the same reasons.